api/HTMLInputElement.json

```diff
@@ -1683,10 +1683,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLInputElement/webkitEntries",
           "support": {
             "chrome": {
-              "version_added": "13"
+              "version_added": "22"
```

Comment:
This is the biggest change in the PR in terms of removing claimed support. Can you check where the Chrome 13 claim came from?

(There are a few other changes that bump `version_added` by one version.)

Reply:
Looks like the original data comes from the wiki migration (in fact, it happens to be my PR, haha!), so I have a feeling that it was inaccurate.

Reply:
Alright, thanks for checking!